airflow/bin/cli.py

```diff
@@ -185,6 +185,56 @@ def trigger_dag(args):
     logging.info(message)
 
 
+def batch_set_pools(args):
+    """
+    set a batch of Airflow pools
+
+    :param args: arguments from argparser
+    :return:
+    """
+    session = settings.Session()
+    pool_names = args.names
+    pool_slots = args.slots
+    pool_descriptions = args.descriptions
+    print("Number of pool names input: {}".format(len(pool_names)))
+    print("Number of pool slots count input: {}".format(len(pool_slots)))
+    print("Number of pool description input: {}".format(len(pool_descriptions)))
```

Comment:
Do we want to log instead of print?

Reply:
If you check Airflow code. It is using print in other functions in cli as well. I am just following its style.

Reply:
Yeah, but immediately reprinting what the user enters seems a bit much. 

I think printing the second set of information is useful though. (and probably should be logged as well)